### PR TITLE
Assignment02

### PR DESCRIPTION
**Pull request by shravan-n**

### CODE REVIEW

File: Assignment02/ContainsAllOrNot.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*Time complexity of this code is O(n) where n is the size of string*/
```
Comment (Seshareddy12) [inline Assignment02/ContainsAllOrNot.java:1]: Mention Space complexity too

**Review by Seshareddy12 — APPROVED** [review]

File: Assignment02/ContainsAllOrNot.java (hunk 1 above)
Comment (Seshareddy12) [reply Assignment02/ContainsAllOrNot.java:1]: Space Complexity is O(1) because , it always constant 26.

Comment (shravan-n) [reply Assignment02/ContainsAllOrNot.java:1]: will change to O(1)

Comment (Seshareddy12) [reply Assignment02/ContainsAllOrNot.java:1]: not required , just to inform u.

**Review by Seshareddy12 — APPROVED** [review]

**Pull request merged**